docs/kb/semgrep-supply-chain/scanning-projects-with-multiple-lock-files.md

```diff
@@ -0,0 +1,108 @@
+---
+tags:
+  - GitHub Actions
+  - Lock files
+description: Scanning projects with multiple lock files in GitHub Actions
+---
+
+# Scanning projects with multiple lock files in GitHub Actions
+
+Your project may have many modules, each with a dependency descriptor file; for example, `pom.xml` if Maven is used as a build dependency tool.
+This [project](https://github.com/finos/legend-engine) complies with the above description.
```

Comment:
What's the purpose of this link? If it's just to show what such a project might look like, think we probably shouldn't link out to random GH projects in our KB/docs for that. If we need an example, creating one we control would be better.

Reply:
You are right. I have already changed the link.